src/components/SignupForm/interfaces/ValidationInterface.tsx

```diff
@@ -0,0 +1,8 @@
+import React, { SetStateAction } from 'react';
```

Comment:
ValidationInterface.tsx  --> ValidationInterfaces.tsx 

Because this file should store more reusable elements in future